framework/material/Dialog.kt

```diff
@@ -40,7 +43,7 @@ import com.vaticle.typedb.studio.state.app.DialogManager
 
 object Dialog {
 
-    val DIALOG_SPACING = 16.dp
+    val DIALOG_SPACING = Theme.DIALOG_PADDING
```

Comment:
Let's make sure that we always propagate our renames to the other symbols whose names are based on them!
```suggestion
    val DIALOG_PADDING = Theme.DIALOG_PADDING
```
But now that these things have the same name, we can simply dissolve this property and just use `Theme.DIALOG_PADDING` directly wherever it was used.
```suggestion
```

Reply:
Done!